torch/utils/_cuda_trace.py

```diff
@@ -3,6 +3,8 @@
 
 from typing_extensions import ParamSpec
 
+_CUDA_HOOK_TRACE_OFFSET = -2
```

Comment:
The change introducing this variable, and its test, could be a separate preliminary PR. The same with the other trace offset variable (either the same or two separate PRs).